lib/node_modules/@stdlib/complex/parse-float64/docs/types/test.ts

```diff
@@ -0,0 +1,70 @@
+/*
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+import parseComplex128 = require( './index' );
+
+
+// TESTS //
+
+// The function parses a string representation of a complex number...
+{
+	const str = '5 + 3.5j';
```

Comment:
These are not unit tests, so splitting out into separate blocks is not necessary. You can consolidate from here to L57 into a single block.